manifests/base/redis/kustomization.yaml

```diff
@@ -8,14 +8,3 @@ resources:
 - argocd-redis-service.yaml
 - argocd-redis-network-policy.yaml
 
-replacements:
```

Comment:
Removed this section as it was creating an `env` section in the `redis-deployment`.